var/spack/repos/builtin/packages/protobuf/package.py

```diff
@@ -138,7 +139,10 @@ def install(self, spec, prefix):
         args = self.cmake_args()
         args.extend(std_cmake_args)
 
-        source_directory = join_path(self.stage.source_path, "cmake")
+        if self.spec.satisfies("@:3.21.7"):
```

Comment:
```suggestion
        if self.spec.satisfies("@:3"):
```
In case they have more releases in the old branch (they seem to be doing this already)